cmd/kubeadm/OWNERS

```diff
@@ -12,14 +12,13 @@ reviewers:
 - fabriziopandini
 - neolit123
 - kad
-- liztio
-- chuckha
 - detiber
 - dixudx
 - rosti
 - yagonobre
-- ereslibre 
-# Might want to add @xiangpengzhao and @stealthybox back in the future
+- ereslibre
```

Comment:
We should also consider grooming @ereslibre for approver imo.

Reply:
+1 from me. should be voted in a meeting? can be added in a separate PR?

also most importantly we should ask @ereslibre if he's ok with that.
at the EOD, reviewers + approvers need to have the bandwidth.

Reply:
It's true that I've been a bit away for some time, but planning to allot some time daily to review PR's, I'd agree if folks want.

Reply:
Let's final vote this week in the meeting then unhold this PR.